inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -14,7 +14,9 @@
 from inductiva.utils.templates import batch_replace_params_in_template
 from inductiva.utils.files import remove_files_with_tag
 
-OPENFOAM_TEMPLATE_INPUT_DIR = "openfoam_template"
+SCENARIO_TEMPLATE_DIR = os.path.join(os.path.dirname(__file__), "..",
```

Comment:
The path to the root of the templates directory should be defined in a dedicated place (e.g. in the templating utils module) for all scenarios to use, otherwise, we will have some duplication. Having to find the templates directory with relative paths everywhere looks a bit error-prone. Wdyt?

Reply:
Seems good!